src/urban_disco/core.clj

```diff
@@ -1,51 +1,8 @@
 (ns urban-disco.core
+  (require [grid.base-grid :as grid-builder]
+           [grid.tile :as tile])
```

Comment:
In all the `ns` forms, you want everything to be fully qualified, i.e. `urban-disco.grid.base-grid`  instead of `grid.base-grid`